proprietary/tokens/src/brand/ams/typography.tokens.json

```diff
@@ -0,0 +1,39 @@
+{
+  "ams": {
+    "brand": {
+      "typography": {
+        "font-family": {
+          "body-text": { "value": "'Amsterdam Sans', Arial, sans-serif" },
```

Comment:
Including `-text` is clearer than just `body`. I dismissed `body-copy` for not being very common. Utrecht uses ‘document’, but headings are also in the document.

> A typesetter […] sees 'body text' as being those sections of the main text that are flowed into columns or justified as paragraphs, as distinct from the headings and any pictures that are floated out of the main body.
– [Wikipedia](https://en.wikipedia.org/wiki/Body_text)